test/index.spec.ts

```diff
@@ -63,6 +64,15 @@ describe("Stylelint Target", () => {
 		await run.stop();
 	});
 
+	it("should support ignorePath", async () => {
+		host.writeMultipleFiles({ ...filesWithErrors, ...gitIgnoreFile });
+		const overrides: Partial<StylelintBuilderOptions> = { ignorePath: "./.gitignore" };
+		const run = await architect.scheduleTarget(stylelintTargetSpec, overrides);
+		const output = await run.result;
+		expect(output.success).toBe(true);
+		await run.stop();
+	});
+
```

Comment:
This test passes, but it also passes when I don't set the path to the `filesWithErrors` `foo.scss` in `gitIgnoreFile`. So I assume, the tests do currently only check if the config is handled correctly. Or did I miss something?